client/src/ocex_commands.rs

```diff
@@ -0,0 +1,228 @@
+use log::*;
+
+use clap::{Arg, ArgMatches, App};
+use clap_nested::Command;
+use my_node_runtime::AccountId;
+use polkadex_sgx_primitives::{AssetId, Balance};
+use sp_core::{sr25519 as sr25519_core, Pair};
+use substrate_api_client::{
+    compose_extrinsic,
+    extrinsic::xt_primitives::UncheckedExtrinsicV4,
+    XtStatus,
+};
+
+use substratee_stf::commands::{common_args_processing, common_args};
+
+pub fn register_account_command<'a>() -> Command<'a, str> {
+        Command::new("register-account")
+            .description("Registers a new main account to the polkadex offchain registry")
+            .options(|app| {
+                app.arg(
+                    Arg::with_name("main")
+                        .takes_value(true)
+                        .required(true)
+                        .value_name("SS58")
+                        .help("Sender's on-chain AccountId in ss58check format"),
+                )
+            })
+            .runner(move |_args: &str, matches: &ArgMatches<'_>| {
+                let chain_api = crate::get_chain_api(matches);
+
+                // get the main account / sender
+                let arg_main = matches.value_of("main").unwrap();
+                let main = crate::get_pair_from_str(matches, arg_main);
```

Comment:
we have to make sure that we update these to `_trusted` when the changes from https://github.com/Polkadex-Substrate/polkadexTEE-worker/pull/84 are merged into develop

Reply:
Or _untrusted. Lets have brenzi first look at #84 to see how he want to differentiate these two keystores.